.github/workflows/make_bundle_conda.yml

```diff
@@ -164,7 +169,7 @@ jobs:
     name: Bundle ${{ matrix.target-platform }}
     runs-on: ${{ matrix.os }}
     needs: packages
-    if: startsWith(github.repository, 'napari')
+    if: startsWith(github.repository, 'napari') && contains((inputs.installer_platforms || 'linux-64,win-64,osx-64,osx-arm64'), matrix.target-platform)
```

Comment:
Because it might not be defined for other triggers. See this other line for a precedent:

https://github.com/napari/packaging/blob/2f43aa7e8358146f3cb4dd6c6b398bd13cfaf5b9/.github/workflows/make_bundle_conda.yml#L65

Reply:
(Hoping it works, idk yet!)

Reply:
Testing at https://github.com/napari/napari/pull/5252

Reply:
but the default value is defined. 

But maybe I do not understand this aspect of GHA.

Reply:
Nah it doesn't work. Conditional matrix entries is not supported like that.

https://stackoverflow.com/questions/65384420/how-to-make-a-github-action-matrix-element-conditional